reporters/cloud-run-scripts/CloudResponse.js

```diff
@@ -38,29 +42,52 @@ app.post("/", async (req, res) => {
       });
     }
 
-    const execResponse = await execShellCommand(req.body.cloudRunCommand, processedEnvironmentVariables);
-    // TODO: refactor this to send a more graceful winston log.
-    console.log("execResponse:", execResponse);
+    const execResponse = await _execShellCommand(req.body.cloudRunCommand, processedEnvironmentVariables);
 
     if (execResponse.error) {
-      console.log("Process exited with error");
       throw execResponse;
     }
-    console.log("Process exited with no error");
+    Logger.debug({
+      at: "CloudRunnerResponse",
+      message: "Process exited with no error",
+      execResponse,
+      childProcessIdentifier: req.body.environmentVariables.BOT_IDENTIFIER
+    });
 
-    res.status(200).send({ message: "Process exited correctly!", execResponse: execResponse });
+    res.status(200).send({ message: "Process exited with no error", execResponse });
   } catch (execResponse) {
-    res.status(400).send({ message: "Process exited with error!", execResponse: execResponse });
+    // If there is an error, send a debug log. We dont want to trigger a `Logger.error` here as this will be dealt with
+    // one layer up in the Hub implementation.
+    Logger.debug({
+      at: "CloudRunnerResponse",
+      message: "Process exited with error",
+      execResponse,
+      jsonBody: req.body,
+      childProcessIdentifier: req.body.environmentVariables.BOT_IDENTIFIER
+    });
+    res.status(400).send({ message: "Process exited with error", execResponse });
   }
 });
 
-function execShellCommand(cmd, inputEnv) {
+function _execShellCommand(cmd, inputEnv) {
   return new Promise((resolve, reject) => {
     exec(cmd, { env: { ...process.env, ...inputEnv } }, (error, stdout, stderr) => {
+      // The output from the process execution contains a punctuation marks and escape chars that should be stripped.
+      stdout = _stripExecOutput(stdout);
+      stderr = _stripExecOutput(stderr);
       resolve({ error, stdout, stderr });
     });
   });
 }
+
+// This Regex removes these and formats the output in a digestible fashion.
```

Comment:
The `execShellCommand` function pipes the output of the `childProcess` `exec` command to: 1) a Winston log and 2) sent back in the HTTP request to the hub. As this captures the response from a console log it contains a bunch of very nasty punctuation and formatting. The screenshot below shows this:
![image](https://user-images.githubusercontent.com/12886084/87419515-e640d000-c5d3-11ea-8a16-328bb2b3d20f.png)

The regex cleans this up to make it a bit more digestible
![image](https://user-images.githubusercontent.com/12886084/87419562-fc4e9080-c5d3-11ea-94dc-0d862c8654ca.png)

The defiantly can be improved but for now it means that _all output logs_ are fed back to the hub for processing. which makes the logic far easier to debug.